### ISSUE TITLE
Model Target Generator application will not start cloud training.

### ISSUE THREAD

**Issue by SynidSweet** [PTCInc/vuforia-engine#17]
**Summarize the bug:**
Model Target Generator application will not start my cloud training.

**Observed problem**
I'm creating an inspection app for industry machinery. The machinery needs to be recognized from a 3D model, so I am trying to set up model recognition with cloud training using a dome guide view. Right now I'm building a mockup, and I've created a 3D model of my home table, uploaded it into the Model Target Generator application, and created a new Model Target using the table. Everything is working fine up til now. The model has status "Guide views set", and "configured for training" status "yes". However, after I press the Train button - I get a popup telling me that I have 20 training sessions left - and I click continue, the train button goes gray and the status says (19 targets remaining, 1 pending) for about 1 second, then everything changes back to excactly the way it looked before and nothing else happens. I still have 20 training sessions left, and I can click the train button as many times as I like. I also left the program on overnight, and nothing changed. I get no error messages.

My model is very simple, it's a table in a single mesh with a very low polygon count and only one material with a single color, no texture.

I've read through anything in the documentation that seems relevant to find out how to approach this, but I have found nothing pertaining to this issue. On google I found one stack overflow question with the same problem, but no clear answer:
https://stackoverflow.com/questions/75978950/why-is-model-target-generator-not-starting-the-training-process

**Affected Vuforia Engine version:**
Model Target Generator v10.14.4.

**Affected platform:**
 - Workstation OS: Windows 11 Home

**Screenshots / Video Recordings**
![image](https://user-images.githubusercontent.com/66042563/233093428-381edc84-799a-4dc5-8422-713cfa7d54e8.png)
![image](https://user-images.githubusercontent.com/66042563/233093489-176d9a71-ed49-449a-9ff4-b03d35cebbeb.png)

**Comment by ptc-ddowning** [comment]
Hello,

Thank you for the report. From the initial report details, this seems symptomatic of two possible issues:

1. The MTG is not able to reach our training services in the cloud. Can you confirm that your firewall has whitelisted `*.vuforia.com`?
2. The 3D model is not compatible with our backend training services

The fact that the failure is immediate and that no training quota has been used makes me believe this is likely related to #1. Please confirm that the MTG is not being blocked by a firewall.

**Comment by SynidSweet** [comment]
Hello again and thank you for the fast response!

The firewall was fine, I'm on a home network, but your second point made me think I should try converting it to other formats. My original upload file was an OBJ file. I exported it as an FBX file instead and it works. It now retains the 1 pending status and the Train button is now a red Cancel button. Thank you! That's one thing that I somehow overlooked.

Either way, it seems like the OBJ didn't cut it. I made no other differences than to just export it as an FBX this time. I made the model in blender. Here are the two files I used in case you want to look at them:
https://drive.google.com/drive/folders/1VjI5q27UxsnJljAjn2NvhL4mVrICuwAl?usp=share_link

BR,
Petter
